src/App.jsx

```diff
@@ -15,9 +15,9 @@ function App() {
       <Routes>
         <Route path="/" element={<Panel isLoggedIn={isLoggedIn}/>}>
           <Route path="/frontpage" element={<FrontPage/>} />
+          <Route path="/teacher/dashboard" element={<TeacherDashboard />} />
```

Comment:
I am of the opinion that we should have one dashboard. If you are a teacher, you should see the courses you manage. If you are a student, you should see the courses you are currently taking, and this component would be at `/dashboard`.

Since this seems to be specifically focused on the teacher side of things, maybe we can leave the student part out until we have a proper login/signup flow and for now we just rename this component.

Reply:
As a further example, it wouldn't make much sense for me to have a route `/teacher/dashboard` but no route `/teacher`. I would expect to only have a child route if the parent exists and it logically makes sense.

Reply:
I agree! I can make those changes in the next PR. I think we can merge this one. We can refactor it how to render it when working Sign Up and Sign In components will be merged, and User Dashboard also.